app/src/main/java/org/stepik/android/domain/banner/model/Banner.kt

```diff
@@ -0,0 +1,37 @@
+package org.stepik.android.domain.banner.model
+
+import com.google.gson.annotations.SerializedName
+
+data class Banner(
+    @SerializedName("type")
+    val type: ColorType,
+    @SerializedName("lang")
+    val language: String,
+    @SerializedName("description")
+    val description: String,
+    @SerializedName("url")
+    val url: String,
+    @SerializedName("screen")
+    val screen: Screen,
+    @SerializedName("position")
+    val position: Int
+) {
+    enum class ColorType(val type: String) {
+        @SerializedName("blue")
+        BLUE("blue"),
+
+        @SerializedName("violent")
+        VIOLET("violet"),
+
+        @SerializedName("green")
+        GREEN("green")
+    }
+
+    enum class Screen(val screen: String) {
```

Comment:
These properties can be extended in future. It will be better to not restrict possible values to have an ability to add new values with backward compatibility. Eg. `screen` = `teach`.